### PR TITLE
Add skade as a core team observer

### PR DESCRIPTION
**Pull request by Manishearth**
r? @skade @steveklabnik @pietroalbini

I'm not sure if I've done the TOML stuff correctly, make sure you wait for Pietro to sanity-check the format before merging.

### CODE REVIEW

File: teams/core-observers.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name = "coore-observers"
+subteam-of = "core"
+
+[people]
+leads = []
+members = [
+    "skade",
+]
+
+[permissions]
```
Comment (pietroalbini) [inline teams/core-observers.toml]: Just a note, the core team has the `bors.rust.review` permission. Not sure if you want to put that here.

File: teams/core-observers.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name = "coore-observers"
```
Comment (pietroalbini) [inline teams/core-observers.toml]: Wrong name.

**Comment by pietroalbini** [comment]
Uuuh, you also need to remove the `bors.rust.permission` from yourself. This wasn't a problem when you first created the other PR, but a change got merged in the meantime that required it.

File: teams/core-observers.toml (hunk 1 above)
Comment (Manishearth) [reply teams/core-observers.toml]: I don't know either :)

**Comment by Manishearth** [comment]
Ah, updated. Though I guess we should allow overlap this way: team memberships can change.

(But eventually I guess we should be creating buckets like journeypeople for these permissions so that individual perms aren't necessary anymore)

File: teams/core-observers.toml (hunk 1 above)
Comment (nikomatsakis) [reply teams/core-observers.toml]: I think it's fine. I would expect that -- like anyone with r+ -- there is an expectation that people will only use it where appropriate. But I could see a core team observer being delegated some task by core team that involves r+.

Comment (Manishearth) [reply teams/core-observers.toml]: Added. Ready to merge, anyone want to push the button?

File: teams/core-observers.toml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+members = [
+    "skade",
+]
+
+[permissions]
+bors.rust.review = true
+
+[website]
+name = "Core Team Observers"
+description = "observers on the core team"
```
Comment (Manishearth) [inline teams/core-observers.toml]: this text could be improved maybe

**Pull request merged**